tests/modules/asset_groups/resources/test_patch_asset_group.py

```diff
@@ -130,34 +130,44 @@ def test_patch_asset_group(
         asset_guid,
     )
 
-    annotation_guid = annot_response.json['guid']
+    annotation1_guid = annot_response.json['guid']
 
-    # Attempt to replace which should fail
-    patch_replace_annot = [utils.patch_replace_op('annotations', annotation_guid)]
-    asset_group_utils.patch_asset_group_sighting(
+    annot_response = annot_utils.create_annotation_simple(
+        flask_app_client,
+        researcher_1,
+        asset_guid,
+    )
+    annotation2_guid = annot_response.json['guid']
+
+    # Attempt to replace which should pass
+    annot_replace_resp = asset_group_utils.patch_asset_group_sighting(
         flask_app_client,
         researcher_1,
         encounter_path,
-        patch_replace_annot,
-        409,
```

Comment:
Don't want to send the expected response code anymore?

Reply:
No, previously this was expected to fail, hence 400 response code, now it is expected to succeed so use the default of 200